### DISCUSSION TITLE
Target build outside of pipeline

### DISCUSSION THREAD

**Discussion by stenw** [ropensci/targets#859] [Help]
I have a target that is expensive to build but I have already done so outside of the pipeline. The results are save as a Rds file. Is there anyway I can 'import' it without rebuilding?

**Comment by wlandau** [comment]
You can treat that RDS file as an input file, like `data.csv` in https://books.ropensci.org/targets/data.html#external-files or `data/raw_data.csv` in https://github.com/wlandau/targets-minimal/blob/c040acc834d96c871b285e7fa145c694332ff164/_targets.R#L7-L15.
